Makefile

```diff
@@ -234,3 +234,7 @@ catalog-push: ## Push a catalog image.
 
 verify: tidy generate
 	git diff --exit-code
+	
+.PHONY: bin/server
+bin/server:
+	go build -o $@ cmd/reporting-operator/main.go
```

Comment:
blocking: We need to rename this cmd/reporting-operator directory to something more appropriate (maybe cmd/server to match the bin/server target?).